core/metacling/src/TClingMethodInfo.cxx

```diff
@@ -449,6 +527,17 @@ int TClingMethodInfo::InternalNext()
             return 1;
       }
 
+      clang::UsingDecl* udecl =
+          llvm::dyn_cast<clang::UsingDecl>(*fIter);
+
+      if ( udecl ) {
+          // A UsingDecl potentially brings in a bunch of functions, so
+          // start an inner loop to catch them all
+          delete fUsingIter;
```

Comment:
What if the `fUsingIter` was pulling in a `UsingDecl`? (i.e. we might need a stack of `UsingIterator`s.) Can you `Error` if `fUsingIter` to see whether that's a problem in reality?